### PR TITLE
RIP36: Colour Coding

### PR DESCRIPTION
**Pull request by AlexWeinstein92**

### CODE REVIEW

**Comment by AlexWeinstein92** [comment]
Screenshot 2024-09-10 at 12 39 11 PM

**Comment by AlexWeinstein92** [comment]
Screenshot 2024-09-10 at 12 44 55 PM

File: src/main/scala/com/ossuminc/riddl/plugins/idea/files/RiddlTokenizer.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "execute", "explained", "field", "fields", "file", "flow", "focus", "foreach", "form",
+    "function", "graph", "group", "handler", "if", "import", "include", "index", "init", "inlet", "inlets",
+    "input", "invariant", "items", "many", "mapping", "merge", "message", "morph", "name", "one", "organization",
+    "option", "optional", "options", "other", "outlet", "outlets", "output", "parallel", "pipe", "plant", "projector",
+    "query", "range", "reference", "remove", "replica", "reply", "repository", "requires", "required", "record",
+    "result", "results", "return", "returns", "reverted", "router", "saga", "schema", "selects", "send", "sequence",
+    "set", "show", "shown", "sink", "source", "split", "state", "step", "stop", "story", "streamlet", "table", "take",
+    "tell", "term", "then", "title", "type", "url", "updates", "user", "value", "void", "when", "where")
+
+  val punctuation: Seq[String] = Seq("asterisk", "comma", "colon", "curlyOpen", "curlyClose", "dot", "equalsSign",
```
Comment (reid-spencer) [inline src/main/scala/com/ossuminc/riddl/plugins/idea/files/RiddlTokenizer.scala:53]: These should all be matching not the text of the name but the special character implied:
* asterisk = *
* comma = ,
* colon = : 
etc.

**Review by reid-spencer — APPROVED** [review]
Generally looks okay. I'd like you to address the punctuation comment and the changes I just pushed to main in RIDDL and then its good to go.

**Comment by AlexWeinstein92** [comment]
Screenshot 2024-09-11 at 1 35 10 PM

**Comment by reid-spencer** [comment]
Looks really good! Merging.

**Pull request merged**